Corona-Warn-App/src/main/res/layout/include_test_result_card.xml

```diff
@@ -31,27 +29,26 @@
 
             <ImageView
                 android:id="@+id/test_result_card_status_icon"
-                android:layout_width="100dp"
-                android:layout_height="100dp"
-                android:background="@{FormatterSubmissionHelper.formatTestStatusIcon(testResult)}"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
                 app:layout_constraintBottom_toBottomOf="parent"
+                android:background="@{FormatterSubmissionHelper.formatTestStatusIcon(testResult)}"
                 app:layout_constraintEnd_toEndOf="parent" />
 
             <TextView
                 android:id="@+id/test_result_card_headline"
```

Comment:
If it's a headline it should use the style `textMultiline`.

Reply:
it does, attribute sorting just moved it down a few lines